packages/cli/src/rpc/explain/collect-location-context.ts

```diff
@@ -39,7 +40,7 @@ export default async function collectLocationContext(
         warn(`[location-context] Skipping location outside source directories: ${location.path}`);
         continue;
       }
-      location.path = location.path.slice(directory.length + 1);
+      location.path = location.path.slice(directory.length + 1) || '.';
```

Comment:
I thought location paths were made always relative in a previous commit?

Reply:
Yes; if the location is absolute path that is the root of a source directory, I put '.' so `path` isn't empty.

Reply:
(I've actually dropped it. Turns out empty path is fine.)